cves/CVE-2015-8217.yml

```diff
@@ -144,10 +157,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: I could not find any unit tests.
```

Comment:
http://coverage.ffmpeg.org/ same kind as before, it seems there's some coverage of the file the vulnerability, so it probably has unit tests